admin/admin-service/src/main/java/io/mosip/admin/dto/FieldDtos.java

```diff
@@ -0,0 +1,17 @@
+package io.mosip.admin.dto;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import java.util.List;
+
+@Data
+@EqualsAndHashCode
+public class FieldDtos {
```

Comment:
why is it named as FieldDtos? why is it plural and also it has fields property in it .. its confusing @dhanendra06

Reply:
Based on the review AdminServiceImpl, I think it is good to rename FieldDtos to SearchFieldsRequestDto.

Reply:
taken care